WebRoot/h/changepass

```diff
@@ -80,63 +83,279 @@
+                                                <fmt:param value="${zimbraPasswordMinDigitsOrPuncs}"/>
+                                            </fmt:message>
+                                            <img src="/img/zimbra/ImgCloseGrayModern.png" id="minDigitsOrPuncsCloseImg" style="display: inline;"/>
+                                            <img src="/img/zimbra/ImgCheckModern.png" id="minDigitsOrPuncsCheckImg" style="display: none;"/>
+                                            ${minDigitsOrPuncsMsg}
+                                        </li>
+                                    </c:if>
+                                </ul>
+                                <label for="confirm" class="zLoginFieldLabel"><fmt:message key="confirmPassword"/></label>
+                                <div class="passwordWrapper">
+                                    <input id="confirm" autocomplete="off" class="zLoginFieldInput" name="loginConfirmNewPassword" type="password" value="" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/>
+                                    <span toggle="#confirm" onClick="showConfirmPassword();" id="confirmShowSpan" style="display: block;"><fmt:message key="show"/></span>
+                                    <span toggle="#confirm" onClick="showConfirmPassword();" id="confirmHideSpan" style="display: none;"><fmt:message key="hide"/></span>
+                                </div>
+                                <div class="signInAndLabel">
+                                    <div>
+                                        <input type="submit" class="loginButton" value="<fmt:message key="save"/>" />
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </c:if>
+                </form>
             </div>
             <div class="decor1"></div>
         </div>
-    </div> 
+    </div>
+    <script>
+        function getElement(id) {
+            return document.getElementById(id);
+        }
+        function check(checkImg, closeImg) {
+            closeImg.style.display = "none";
+            checkImg.style.display = "inline";
+        }
+        function unCheck(checkImg, closeImg) {
+            closeImg.style.display = "inline";
+            checkImg.style.display = "none";
+        }
+        function resetImg(condition, checkImg, closeImg){
+            condition ? check(checkImg, closeImg) : unCheck(checkImg, closeImg);
+        }
+        function showPassword(x, y, z) {
+            if (x.type === "password") {
+                x.type = "text";   
+                y.style.display = "none";
+                z.style.display = "block";
+            } else {
+                x.type = "password";
+                y.style.display = "block";
+                z.style.display = "none";
+            }
+        }
+        function showOldPassword() {
```

Comment:
where are we doing match of old password with new password?

Reply:
Fixed.